src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+        method="post"
+          action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+            data-qa="big"
+    >
```

Comment:
go through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and fix your codestyle, attributes shouldn't look like that, indents should be consistent, fix it for all tags
```suggestion
    <form
      class="search-form"
      method="post"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      data-qa="big"
    >
```

Reply:
done